Update icon registry links in documentation (#7180)
umbraco/UmbracoDocs: 14/umbraco-cms/customizing/foundation/icons.md

# Icons

> The icons from the Umbraco backoffice are based on [Lucide Icons](https://lucide.dev/). The syntax for getting the icons starts with`icon-`. You can find the list of all icons in the [Icon registry list on GitHub](https://github.com/umbraco/Umbraco-CMS/tree/main/src/Umbraco.Web.UI.Client/src/packages/core/icon-registry/icons).

Reviewer: ```suggestion
The icons in the Umbraco backoffice are based on [Lucide Icons](https://lucide.dev/). The syntax for using an icon starts with `icon-`. You can find the full list of available icons in the [Icon registry list on GitHub](https://github.com/umbraco/Umbraco-CMS/tree/main/src/Umbraco.Web.UI.Client/src/packages/core/icon-registry/icons).
```
